lisa/tools/nfs_server.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+from typing import List
+
+from lisa.executable import Tool
+from lisa.operating_system import SLES, Debian, Redhat
+from lisa.tools import Echo, Firewall
+from lisa.tools.service import Service
+from lisa.util import UnsupportedDistroException
+
+
+class NFSServer(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def can_install(self) -> bool:
+        return True
+
+    def setup(self, client_ips: List[str], dir_name: str) -> None:
```

Comment:
how about call it `create_shared_dir`? It's clearer than "setup".

Reply:
This step also restarting nfs server and updating nfs exports file . I thought of using name `setup_server` but `_server` postfix is already present in tool name.

Reply:
The setup is near to "install", which is in the super class already. From the functionality, it passes in the clients and dir name, so it's more like to create/setup a shared dir, not the server.

Reply:
I can rename it to `create_shared_dir` then